test/test-utils/gameManager.ts

```diff
@@ -328,7 +328,7 @@ export class GameManager {
       "SelectModifierPhase",
       UiMode.MODIFIER_SELECT,
       () => {
-        const handler = this.scene.ui.getHandler() as ModifierSelectUiHandler;
+        const handler = this.scene.ui.getCurrentHandler() as ModifierSelectUiHandler;
```

Comment:
NIT: Use generic param `<T>`

```suggestion
        const handler = this.scene.ui.getCurrentHandler<ModifierSelectUiHandler>();
```